packages/app/public/workflow-editor-envelope/__NOTE__.txt

```diff
@@ -0,0 +1,4 @@
+This folder (`public/workflow-editor-envelope`) must not be pushed to source control.
```

Comment:
Maybe we can add this to `.gitignore` to avoid issues

Reply:
If we add it to gitignore we won't ever see "unwanted" changes in these files.